src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/EnumToBoolConverter.shared.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Xamarin.CommunityToolkit.Extensions.Internals;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	/// <summary>
+	///     Convert an <see cref="Enum" /> to corresponding <see cref="bool" />
+	/// </summary>
+	public class EnumToBoolConverter : ValueConverterExtension, IValueConverter
+	{
+		/// <summary>
+		///     Enum values, that converts to <c>true</c> (optional)
+		/// </summary>
+		public IList<Enum> TrueValues { get; } = new List<Enum>();
```

Comment:
Can we replace "Enum" with "object" and make this converter working for any type?

Int to bool, double to bool, object to bool etc.

Reply:
@AndreiMisiukevich Ok. Will try this tomorrow.

Reply:
@AndreiMisiukevich Wouldn't it be better, to have another converter, that handle this problem, like `IsInListConverter` or `IsInCollectionConverter`. This can be implemented in general (object) and generic way. I can implement this, but would prefer to do this in new PR (and corresponding issue).

The `EnumToBoolConverter` uses some enum specifics (like flags).